STYLEGUIDE.md

```diff
@@ -0,0 +1,306 @@
+
+### Do
+
+“You can enable triggers for the Storage Category with Amazon S3 & Amazon DynamoDB as Providers. Amplify supports associating Lambda triggers with S3 and DynamoDB events.”
+* * *
+
+## 13. Lead with location; end with action
+
+When instructing a learner on how to do something, start with the location of the action they’re going to perform so they can navigate to that place, then end with the action they can then use.
+
+### Don’t
+
+“Select **Edge optimized** in the **Endpoint Type** drop-down.”
+
+### Do
+
+“In the **Endpoint Type** drop-down, select **Edge optimized.”**
+* * *
+
+## 14. Use monospace fonts for code, commands, copy-ables; use bold for menus, URLs (clickable), and other UI copy
+
+Code tags and monospace fonts are used to represent code that goes into files or CLI. UI copy, file names, URLs you can click on use **bold.** Remember: monospace is for anything that lives in or is copied to a file or a command line. Everything else is bold.
+
+### Don’t
+
+“To create a new file, go to ‘Menu’ and click on `new file`.”
+
+“In your browser, open `http://localhost:3000/` ”
+
+“Enter the Amplify Library for Swift GitHub repo URL (`https://github.com/aws-amplify/amplify-swift`) into the search bar and hit **Enter**.”
+
+### Do
+
+“To create a new file, go to **Menu > New File.”**
+
+“In your browser, open **http://localhost:3000/”**
+
+“Enter the Amplify Library for Swift GitHub repo URL (`https://github.com/aws-amplify/amplify-swift`) into the search bar and hit **Enter**.”
+* * *
+
+## 15. Use carets to help users navigate menus; use “select” instead of “click”
+
+* For instructions for navigating a menu, use carets to indicate the hierarchy of the menu items. This is concise and easy to follow.
+* Tell people to “select” things instead of “click on them”. Not everyone uses a mouse—many people tab or touch items.
+
+
+### Don’t
+
+“To create a new file, go to ‘Menu,’ select ‘Art,’ and click on ‘new file’.”
+
+“Click on ‘upload image.’”
+
+### Do
+
+“To create a new file, go to **Menu > Art > New File**.”
+
+“Select **upload image**.”
+* * *
+
+## 16. Bracket `<replacement text>` in codeblocks
+
+In code blocks, refer to placeholder text with angle brackets, lowercase, with dashes. 
+
+### Don’t
+
+* `amplify/backend/api/~apiname~/schema.graphql`
+* `amplify/backend/api/YOUR-API-NAME/schema.graphql`
+* `amplify/backend/api/<YOUR_API_NAME>/schema.graphql`
+* `amplify/backend/api/<your_api_name>/schema.graphql`
+* `amplify/backend/api/<your api name>/schema.graphql`
+* `amplify/backend/api/{yourAPIname}/schema.graphql`
+
+### Do
+
+**`amplify/backend/api/<api-name>/schema.graphql`**
+* * *
+
+## 17. Refer to Amplify files in the /amplify directory
+
+When referring to files in the **amplify** directory, always preface with **amplify/**
+
+### Don’t
+
+**`backend/api/<api-name>/schema.graphql`**
+
+### Do
+
+* * *
```

Comment:
For #17 here, I think the "Do" item is missing here to compare with the "Don't" above of:

**`backend/api/<api-name>/schema.graphql`**